src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import static core.basesyntax.ColorSupplier.getRandomColor;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+
+        System.out.println(ball1);
+        System.out.println(ball2);
+        System.out.println(ball3);
+        System.out.println();
+
+        getRandomColor();
```

Comment:
This call to getRandomColor() is not being used. If you need a color, you should use the Lottery instance to get a random Ball which includes a color.